vietnamese/kechuyen.md

```diff
@@ -14,4 +14,6 @@ Tôi biết Tiền Giang rất đẹp, Quê tôi thì ở [Đà Nẵng](DaNang/D
 
 Nhưng hiện tại thì tôi rời xa Đà Nẵng từ nhỏ và đang sống ở [Sài Gòn](SaiGon/SaiGon.md)
 
-Nhắc tới Sài Gòn, không thể không nhắc tới [Cà phê bệt](https://www.youtube.com/watch?v=avJk5KnEpaY) nhỉ?
\ No newline at end of file
+Nhắc tới Sài Gòn, không thể không nhắc tới [Cà phê bệt](https://www.youtube.com/watch?v=avJk5KnEpaY) nhỉ?
+
+Tôi thì chưa từng đến Sài Gòn, tôi là người [Hà Lội gốc](haloi/toi_la_nguoi_ha_loi_goc.md), rể [Hải Phòng](Hai-Phong/Hai_Phong_thanh_pho_cang.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.